addons-l10n/pl/search-my-stuff.json

```diff
@@ -1 +1 @@
-{"search-my-stuff/project-placeholder":"Szukaj projektow","search-my-stuff/studio-placeholder":"Szukaj studia","search-my-stuff/search-by":"Szukaj przez","search-my-stuff/progress-header":"Wyszukiwanie...","search-my-stuff/@description":"Umożliwia przeszukiwanie projektów, studiów i śmietnika na stronie Moje rzeczy. Po prostu zacznij wpisywać tekst w dowolnym miejscu na stronie, aby rozpocząć wyszukiwanie. Po zakończeniu wyszukiwania można także użyć kombinacji klawiszy Ctrl+Enter, aby szybko otworzyć górny wynik wyszukiwania.","search-my-stuff/@info-pleaseBePatient":"Jeśli masz wiele projektów lub studiów, wyszukiwanie może trochę potrwać."}
```

Comment:
what

Reply:
idk

Reply:
i didn't mean to do that

Reply:
prolly git fuckery when i was trying to roll back a commit

Reply:
fixed now